instructors.md

```diff
@@ -426,85 +427,70 @@ and have a rubric to follow.
+    *   Send them a link to the debriefing Etherpad.
+
+### During
 
 *   Remind students that the Etherpad is a public, persistent document,
     and that ideas and issues from the Etherpad may be used by the SCF.
-
-*   Explain how the checkout procedure itself goes:
-
-    *   After the trainee has completed the discussion session to the discussion leader's satisfaction,
-        the leader will forward her name to the trainer.
-
-    *   The trainer will then schedule a 5-minute slot for a demonstration lesson
-        and communicate it to the trainee.
-
-    *   In that session,
-        the trainee will be asked to teach 5 minutes from her chosen lesson
-        selected by the trainer.
-        The trainee will *not* know in advance which 5 minutes she will be asked to teach
-        (which encourages her to prepare the entire lesson).
-
-    *   Everyone will be notified afterwards by email whether they
-        passed or not.
-
+*   Briefly review the [checkout procedure](checkout.html).
+    Make sure trainees understand that *they* pick the lesson,
+    but *we* pick the segment from that lesson.
+*   Inform them that they will be notified after the session by email
+    about whether they passed.
 *   Make sure that all know where the lesson material can be found.
     Ensure that they all know about the instructor guide.
+*   Discuss the fact that the lesson material is under continuous development.
+    Tell the students that any and all changes to the lesson material that might improve on it is welcome.
+*   Ask the leading questions give in [the rubric in the checkout description](checkout.html#rubric).
 
-*   Discuss the fact that the lesson material is under continuous
-    development.  Tell the students that any and all changes to the
-    lesson material that might improve on it is welcome.  If anybody
-    has any comments on the material, they are very welcome.
-
-*   Go through each of the modules in the lesson and ask if anybody has
-    any problems, questions, or comments about the module.  This
-    walkthrough is particularly important if any of the participants
-    are on the quieter side.
+Note:
 
-*   It is important that we ensure that all of the participants get to
-    discuss their issues.  In cases where all participants are vocal,
-    a structured walkthrough may not be necessary.
+1.  It is important that we ensure that *all* trainees get to raise issues.
+    If some trainees are more vocal than others,
+    or if trainees are mixed with experienced instructors (in debriefing sessions),
+    the discussion leader must moderate and ask leading questions
+    to give everyone a fair chance to speak.
+2.  We do not need to ask every question of every trainee,
+    just as we do not ask every trainee to teach an entire half-day lesson.
+    Try instead to mix up the questions to check that the trainees have prepared adequately.
 
```

Comment:
What I've tried is starting with a "round table". I pick an open-ended question from the rubric and asked the trainees one after another (I said we'd do "round table"). Then (depending on the group) the discussion session becomes more free but the leader still needs to act as the moderator. I think the key is to ask directly those who tend to be quiet.